aiohttp/abc.py

```diff
@@ -146,3 +146,27 @@ def write_eof(self, chunk=b''):
     @abstractmethod
     def drain(self):
         """Flush the write buffer."""
+
+
+class AbstractAccessLogger(ABC):
+
+    def __init__(self, logger, log_format):
+        self.logger = logger
+        self.log_format = log_format
+
+    @abstractmethod
+    def _log(self, request, response, time):
+        """
+
+        :param request: Request object.
```

Comment:
Please don't use sphinx formatting in docstrings -- it is a text for readers.
I'd like to rip it out everywhere and use only hand written documentation.